<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuário entrando</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(253, 254, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="login">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>110</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_entrar">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 233, 19);
border-radius: 15px;
border-style: outset;
border-width: 0px;</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_cadastre">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>380</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 163, 2);
border-radius: 10px;
border-style: outset;
border-width: 0px;</string>
    </property>
    <property name="text">
     <string>Cadastre-se agora!</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_semCadatro">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>380</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(114, 75, 17);</string>
    </property>
    <property name="text">
     <string>Não está cadastrado?</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_erro">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>280</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imagem_login">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>110</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Perfil.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="imagem_senha">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>175</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Chave.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="senha">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>175</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="by">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>400</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>by antthLuca</string>
    </property>
   </widget>
   <widget class="QLabel" name="instrucao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preencha os campos abaixo e clique no botão para entrar:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
